src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1169,6 +1173,12 @@ def dbapi(
                 For example, `"SET TRANSACTION ISOLATION LEVEL READ UNCOMMITTED"` can be used in SQL Server
                 to avoid row locks and improve read performance.
                 The `session_init_statement` is executed only once at the beginning of each partition read.
+            batch_size: The number of fetched batches to combine into a single Parquet file.
```

Comment:
I feel like maybe the name of this parameter is not clear enough?
for example, we have batch_size of 1 and fetch_size of 1000, I would think the cap_size of 1000 is 'size of a batch', while the batch_size here is more like 'number of fetches'.
what do you think?

Reply:
updated to `fetch_merge_count`